model/Core/Properties/typeFromSource.md

```diff
@@ -0,0 +1,17 @@
+SPDX-License-Identifier: Community-Spec-1.0
+
+# typeFromSource 
+
+## Summary
+
+typeFromSource is a value used to define an item within the DefinitionSource.
```

Comment:
```suggestion
typeFromSource is a value used to define an item within the definitionSource.
```

Should `DefinitionSource` be `definitionSource`?

I cannot find `DefinitionSource` class, but there is `definitionSource` in Core.

definitionSource.md